### PR TITLE
Removes Deformed Sprites

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Removes deformed sprites from the game.

These are dumb, and no one *ever* notices them---not to mention a good percentage of the species don't even have them.

Making two sets of sprites to be used for every species, just for the rare times they have clone damage is patently ridiculous---especially when people are wearing clothing most of the time, and, even if they're not, the changes that happens with this are *barely* noticeable; I'd venture to guess a good majority of our players didn't even know this *existed*.

:cl: Fox McCloud
del: removes deformed sprites from having high clone damage
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species/plasmaman.dm [hunk 1]
```diff
@@ -2,7 +2,6 @@
 	name = "Plasmaman"
 	name_plural = "Plasmamen"
 	icobase = 'icons/mob/human_races/r_plasmaman_sb.dmi'
-	deform = 'icons/mob/human_races/r_plasmaman_pb.dmi'  // TODO: Need deform.
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/species/plasmaman.dm:5]: This is the only one I'm keeping; we need to standardize on one or the other.

Quite frankly, the "deformed" (it's not) one is better, but I'll save that for another PR.

File: code/modules/mob/living/carbon/human/species/diona.dm [hunk 2]
```diff
@@ -2,7 +2,6 @@
 	name = "Diona"
 	name_plural = "Dionaea"
 	icobase = 'icons/mob/human_races/r_diona.dmi'
-	deform = 'icons/mob/human_races/r_def_plant.dmi'
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/species/diona.dm:5]: This one has a totally different look, to boot.

**Review by SteelSlayer — APPROVED** [review]

**Review by dearmochi — APPROVED** [review]

**Comment by Carthusia** [comment]
God i only recently saw this recently, they are horrifying and I love them.
Perhaps we work to add more sprites like this?

**Review by Kyep — APPROVED** [review]

**Comment by Fox-McCloud** [comment]
>Perhaps we could work to add more sprites like this?

No.

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**